ynr/apps/candidates/tests/test_changes_for_review.py

```diff
@@ -230,25 +308,26 @@ def setUp(self):
                 self.la_id = la.pk
 
     def test_needs_review_as_expected(self, mock_datetime):
+        self.maxDiff = None
         mock_datetime.now.return_value = self.current_datetime
-        needs_review_dict = LoggedAction.objects.in_recent_days(
-            5
-        ).needs_review()
+        needs_review_qs = (
+            LoggedAction.objects.in_recent_days(5)
```

Comment:
Why are we looking at `in_recent_days(5)` here but `in_recent_days(1)` in `feeds.py`?

Reply:
🤷‍♂ It was like that when I got here